test/Conversion/tritongpu_to_llvm.mlir

```diff
@@ -48,11 +51,26 @@ module attributes {"triton_gpu.num-warps" = 4 : i32} {
 module attributes {"triton_gpu.num-warps" = 1 : i32} {
   // CHECK-LABEL: vectorized_load_f16
   func @vectorized_load_f16(%a_ptr_init: tensor<256x!tt.ptr<f16>, #blocked0>, %cst : tensor<256xi1, #blocked0>, %cst_0 : tensor<256xf16, #blocked0>) {
-    // CHECK: llvm.inline_asm
-    // GCN-SAME: global_load_ushort $0, $1, off offset:0
+    // GCN-NOT: llvm.inline_asm
+    // GCN: llvm.load {{.*}} : !llvm.ptr<f16, 1>
+    // GCN: llvm.load {{.*}} : !llvm.ptr<f16, 1>
+    // GCN: llvm.load {{.*}} : !llvm.ptr<f16, 1>
+    // GCN: llvm.load {{.*}} : !llvm.ptr<f16, 1>
+    // GCN: llvm.load {{.*}} : !llvm.ptr<f16, 1>
+    // GCN: llvm.load {{.*}} : !llvm.ptr<f16, 1>
+    // GCN: llvm.load {{.*}} : !llvm.ptr<f16, 1>
+    // GCN: llvm.load {{.*}} : !llvm.ptr<f16, 1>
```

Comment:
You can use `GCN-COUNT-` directive (e.g. `GCN-COUNT-8 llvm.load {{.*}} : !llvm.ptr<f16, 1>`) here and in the same places: https://llvm.org/docs/CommandGuide/FileCheck.html#the-check-count-directive.

Reply:
This is useful, thank you!